workflow/defaults/default_resources.yaml

```diff
@@ -36,7 +36,7 @@ godas_resource_table: !Select
       r3dvar:       [  400,   40, !timedelta "01:30:00", 1,   "3072" ]
       r3denvar:     [  240,   40, !timedelta "01:00:00", 1,   "3072" ]
       da_prep:      [    1,    1, !timedelta "00:30:00", 1,   "3072" ]
-      obs_prep:     [   12,   12, !timedelta "00:10:00", 1,   "3072" ]
+      obs_prep:     [   12,   12, !timedelta "01:30:00", 1,   "3072" ]
```

Comment:
Why do we need 1.5 hours?

Reply:
@flampouris, 1.5 hrs was added for the raw data to be pre-processed. for HofX, it is not neede as we already the pre-processed data.
I can change it to10 mins as previous.

Reply:
@flampouris changed the obs_prep time for  Ocean025deg to 10 mins.